sellotape/sellotape_dj/main_app/views.py

```diff
@@ -1,4 +1,45 @@
 from django.shortcuts import render
+import datetime
+from.models import Stream, User
+# pip install --upgrade google-api-python-client
+from apiclient.discovery import build
+
+api_key = "AIzaSyCAL9HUYTpY9vX05h2Wnk5CAmYmPvSxX8E"
+youtube = build('youtube', 'v3', developerKey=api_key)
 
 def index(request):
     return render(request, 'sellotape/index.html')
+	
+def get_channel_videos(request):
```

Comment:
As @ifireball noted, this shouldn't be a view.. as I see it, views correspond to an HTTP request and response related to some page the user is trying to access. This is more a background process to fill Streams data.